estuary/utils/story.py

```diff
@@ -185,6 +188,13 @@ def get_wait_times(self, results):
 
             next_artifact_start_time = datetime.strptime(next_artifact.timeline_timestamp,
```

Comment:
Optional: It seems a bit clunky to convert a `datetime` object to a string in the `timeline_timestamp` property method and then converting right back to a `datetime` object. It might be cleaner to have a second property method that returns a `datetime` object and then `timeline_timestamp` calls that new property method, or you can just have `timeline_timestamp` return a `datetime` object.